webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/services/policies/NodeConfigData.scala

```diff
@@ -272,65 +280,66 @@ ootapja6lKOaIpqp0kmmYN7gFIhp
   val rootHostname = "server.rudder.local"
   val rootAdmin    = "root"
 
-  val rootNode = Node(
+  val factRoot = CoreNodeFact(
```

Comment:
important change in test data creation in `NodeConfigDate`: now, the base data is a  `CoreNodeFact`, and `Node` and `NodeInfo` are derived from it.